packages/landing/src/components/map.tsx

```diff
@@ -128,7 +138,7 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
   }
 
   render(): ComponentChild {
-    const isLayerSwitcherEnabled = Config.map.tileMatrix === GoogleTms && !Config.map.isDebug;
+    const isLayerSwitcherEnabled = false; //Config.map.tileMatrix === GoogleTms && !Config.map.isDebug;
```

Comment:
Might be good just remove this variable and remove the check after?

Reply:
ahh was testing something, good find.